app/FrameDataExtractor.py

```diff
@@ -57,7 +57,10 @@ def getImageForPhrase(phrase):
         'offset': '0',
         'mkt': 'en-in',
         'safeSearch': 'Moderate',
-        'imageType': 'Clipart'
+        'imageType': 'Clipart',
+        'size': 'medium',
+        'width': 400,
```

Comment:
As of now lets keep it this. But we need to figure out the optimal size based on image type and associated key phrase.